CHANGELOG.md

```diff
@@ -18,6 +19,7 @@
 ## New Features
 - Initial RAFT version
 - PR #3: defining raft::handle_t, device_buffer, host_buffer, allocator classes
+- PR #12: Spectral Clustering
```

Comment:
This is already under 0.15, it should be removed from 0.14 log

Reply:
Looks like this already got changed as requested. Resolving, then.

Reply:
Not sure the change went through, I still see line 22 as a duplicate of line 4

Reply:
Removed the duplicate.